compiler/rustc_const_eval/src/transform/check_consts/check.rs

```diff
@@ -407,6 +390,7 @@ impl<'mir, 'tcx> Checker<'mir, 'tcx> {
                     | ty::PredicateKind::ConstEvaluatable(..)
                     | ty::PredicateKind::ConstEquate(..)
                     | ty::PredicateKind::OpaqueType(..)
+                    | ty::PredicateKind::Trait(..)
```

Comment:
this whole function is now a noop, isn't it?

Reply:
Good catch. I removed it.